libcaliptra/examples/generic/main.c

```diff
@@ -0,0 +1,63 @@
+// Licensed under the Apache-2.0 license
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+#include <unistd.h>
+
+#include "caliptra_api.h"
+
+// Interface defined values
+extern struct caliptra_fuses  fuses;        // Device-specific location of Caliptra fuse data
+extern struct caliptra_buffer image_bundle; // Device-specific location of Caliptra firmware
+
+int main(int argc, char *argv[])
+{
+    int status;
+
+    fuses = (struct caliptra_fuses){0};
+
+    if ((status = caliptra_init_fuses(&fuses)) != 0)
+    {
+        printf("Failed to init fuses: %d\n", status);
+        return status;
+    }
+
+    // Initialize FSM GO
+    caliptra_bootfsm_go();
+    
+    // Wait until ready for FW
+    caliptra_ready_for_firmware();
+
+    // Load Image Bundle
+    // FW_PATH is defined on the compiler command line
+    caliptra_upload_fw(&image_bundle);
+
+    uint32_t FIPS_VERSION = 0x46505652;
```

Comment:
Should commands be part of the API rather than defined as part of the test?

Reply:
Fabrizio wants to encapsulate all of these in an enum in a future PR.

Reply:
Doesn't need to be addressed in this commit, just thinking about it since the command definitions aren't going to differ by vendor.

Reply:
Yeah, the work is far to be completed...plan was to merge this as it has some dependency on the HW Model...then we can start adding more commands/definitions in the API...